### ISSUE TITLE
Slidev not work in monorepo

### ISSUE THREAD

**Issue by yjl9903** [slidevjs/slidev#363]
**Describe the bug**
Slidev in a monorepo is not working.

When I start the dev server, it displays:

```
Could not determine entry point from rollupOptions or html files. Skipping dependency pre-bundling.
```

When I build the slides, it displays:

```
[vite]: Rollup failed to resolve import "vue" from "..\..\..\..\..\..\@slidev\slides\2.md".
This is most likely unintended because it can break your application at runtime.
If you do want to externalize this module explicitly add it to
`build.rollupOptions.external`
```

**To Reproduce**
Simple reproduce repo [slidev-monorepo](https://github.com/yjl9903/slidev-monorepo).

**Desktop (please complete the following information):**
 - OS: Windows 10 21H1
 - Browser: Edge
 - Slidev version: 0.27.2

**Comment by antfu** [comment]
The first warning is a false alarm, and fixed in 0.28

Slidev require pnpm to work in hoist mode (or you will need to explicit list all sub deps). In your case move your .npmrc to root should work.

**Comment by yjl9903** [comment]
> The first warning is a false alarm, and fixed in 0.28
> 
> Slidev require pnpm to work in hoist mode (or you will need to explicit list all sub deps). In your case move your .npmrc to root should work.

Thanks. It works.

And I think we can provide more guides about integrating it into your existing projects at docs [Installation](https://sli.dev/guide/install.html#install-manually) :smile:.
